### ISSUE TITLE
Accesibility of Liverhelperchat (WCAG)

### ISSUE THREAD

**Issue by black23** [LiveHelperChat/livehelperchat#1771]
In the EU, there is a fictitious registration dedicated to the accessibility of websites and apps. The LHC has several shortcomings. It would be possible and get them in line with [WCAG](https://www.w3.org/TR/WCAG21/).
So far I've come across these:

- Missing form label: if a text label for a form control is visible, use the <label> element to associate it with its respective form control. If there is no visible label, either provide an associated label, add a descriptive title attribute to the form control, or reference the label(s) using aria-labelledby. Labels are not required for image, submit, reset, button, or hidden form controls.
- If is chat in window the page has no headings. Headings (<h1>-<h6>) provide important document structure, outlines, and navigation functionality to assistive technology users.

Maybe I missed something, it would be great if accessibility was addressed during development. I'd be happy for further suggestions in this area.

**Comment by black23** [comment]
![obrazek](https://user-images.githubusercontent.com/5516936/174553712-df02ce73-3532-4539-8390-a04bdf51f474.png)
Purpose of of [non text link](https://www.w3.org/TR/WCAG21/#dfn-programmatically-determinable).

[Bypass block](https://www.w3.org/WAI/WCAG21/Techniques/html/H64): title of iframe

**Comment by remdex** [comment]
Please create a pull request with these changes :)

**Comment by black23** [comment]
I'm just user, not programmer. Sorry.

**Comment by remdex** [comment]
As these are UX and I'm overloaded with paid support, these can be implemented faster sponsored way :) otherwise you will have to wait quite long :D
